### ISSUE TITLE
Possible to disallow initial value?

### ISSUE THREAD

**Issue by Donaldini** [sasstools/sass-lint#1019]
Is it possible to disallow the use of certain values?

My specific use case is the use of the _initial_ value inside e.g. the _color_ property.
I want to disable this because this value is [not supported by IE](http://caniuse.com/#feat=css-initial-value), but seems like an excellent fix for some issues (until you test in IE).

`color: initial // Does not work on IE`

**What version of Sass Lint are you using?** 1.10.2

**Comment by DanPurdy** [comment]
There's a PR open in #1142 which will add a rule to do this.

**Comment by agnoster** [comment]
Though notably #1142 only disallows values for a specific property - a blanket blacklist for all instances of the use of the `initial` value across all property names isn't possible. But it could be extended to allow e.g.

```yaml
no-disallowed-property-values:
  properties:
    '*': initial
```
